README.md

```diff
@@ -1,3 +1,61 @@
-# Sonor
+# Sonor app
 
-Case management web application for administration of Computer-Assisted Personal Interviewing (CAPI)
+## Installation
+### Requirements
+
+- Node.js
+- npm 
+
+### Install dependencies
+
+`npm install`
+
+
+## Usage
+
+
+
+### Start the app
+
+`npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### Start mock servers
+
+To test the application at this point you need to use the server *mocked_apis_server* to serve the pearlJam mocked APIs.
+
+Go to the *mocked_apis_server* folder and run
```

Comment:
Can't find folder for running mocked api

Reply:
Done. 
mocked_apis_server.zip added at root project